packages/core/src/to-json-schema.ts

```diff
@@ -0,0 +1,698 @@
+import type * as checks from "./checks.js";
+import type * as JSONSchema from "./json-schema.js";
+import { $ZodRegistry, globalRegistry } from "./registries.js";
+import type * as schemas from "./schemas.js";
+
+interface JSONSchemaGeneratorParams {
+  /** A registry used to look up metadata for each schema. Any schema with an `id` property will be extracted as a $def. */
```

Comment:
this option has default value according to the implementation below

```suggestion
  /** 
   * A registry used to look up metadata for each schema. Any schema with an `id` property will be extracted as a $def.
   * @default globalRegistry
   */
```

Reply:
https://github.com/colinhacks/zod/pull/4223